exercises/practice/queen-attack/.meta/queen_attack.example

```diff
@@ -1,12 +1,21 @@
-create((X,Y)) :- X >= 0, Y >= 0, 7 >= X, 7 >= Y.
+% Please visit https://exercism.io/tracks/prolog/installation
+% for instructions on setting up prolog.
+% Visit https://exercism.io/tracks/prolog/tests
+% for help running the tests for prolog exercises.
 
-column((_, Y), (_, V)) :- Y =:= V.
+% Replace the goal below with
+% your implementation.
```

Comment:
We don't need this comment in the example file, it's okay to remove.

Reply:
> We don't need this comment in the example file, it's okay to remove.

removed and committed.